lib/logcache/client.rb

```diff
@@ -44,8 +44,14 @@ def container_metrics(source_guid:, start_time:, end_time:, envelope_limit: DEFA
 
     private
 
-    def with_request_error_handling(_source_guid)
+    def with_request_error_handling(source_guid)
       tries ||= 3
+      start_time = Time.now
+      time_taken = (Time.now - start_time) * 1000 # convert to milliseconds to get more precise information
+      logger.info("Response time: #{time_taken.round(2)} ms",
+                  { source_id: source_guid,
+                    event: 'log_cache_request',
+                    time_taken_in_ms: time_taken.round(2) })
```

Comment:
```suggestion
                    response_time_ms: response_time_ms })
```
Just personal naming preference.